### PR TITLE
Update testing harness

### PR DESCRIPTION
**Pull request by confusdcodr**

### CODE REVIEW

File: modules/users/README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
...
-| tags | Map of tags to apply to the IAM roles | map | `<map>` | no |
-| template\_paths | Paths to the directories containing the templates for IAM policies and trusts | list | n/a | yes |
-| template\_vars | Map of input variables for IAM trust and policy templates | map | `<map>` | no |
-| users | Schema list of users, consisting of user name, policy path, and permission boundary, policy name, and policy path | list | `<list>` | no |
+| create\_policies | Controls whether to create IAM policies | bool | `"true"` | no |
+| create\_users | Controls whether an IAM user will be created | bool | `"true"` | no |
+| tags | Map of tags to apply to the IAM roles | map(string) | `<map>` | no |
+| template\_paths | Paths to the directories containing the templates for IAM policies and trusts | list(string) | n/a | yes |
+| template\_vars | Map of input variables for IAM trust and policy templates | map(string) | `<map>` | no |
+| users | Schema list of users, consisting of `name`, `policy_path`, and `permission_boundary`, `policy_name`, and `inline_policy` | list | `<list>` | no |
```
Comment (lorengordon) [inline modules/users/README.md]:
```suggestion
| users | Schema list of users, consisting of `name`, `policy_path`, `permission_boundary`, `policy_name`, and `inline_policy` | list | `<list>` | no |
```

File: modules/roles/README.md [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
-| create\_instance\_profiles | Controls whether to create IAM instance profiles | string | `"false"` | no |
-| create\_policies | Controls whether to create IAM policies; when false, `policy` must be an ARN | string | `"true"` | no |
-| create\_roles | Controls whether to create IAM roles | string | `"true"` | no |
+| create\_instance\_profiles | Controls whether to create IAM instance profiles | bool | `"false"` | no |
+| create\_policies | Controls whether to create IAM policies; when false, `policy` must be an ARN | bool | `"true"` | no |
+| create\_roles | Controls whether to create IAM roles | bool | `"true"` | no |
 | max\_session\_duration | The maximum session duration (in seconds) for each role, value must be between 60 to 43200 seconds | string | `"43200"` | no |
 | roles | Schema list of IAM roles, consisting of name, policy template, and trust template | list | `<list>` | no |
```
Comment (lorengordon) [inline modules/roles/README.md]: May also well also update this description, similar to the `users` update

File: modules/users/README.md (hunk 1 above)
Comment (confusdcodr) [reply modules/users/README.md]: Fixed

File: modules/roles/README.md (hunk 2 above)
Comment (confusdcodr) [reply modules/roles/README.md]: Fixed

**Pull request merged**